Cargo.toml

```diff
@@ -20,7 +24,10 @@ tempfile = "3.0"
 [target.'cfg(target_os = "windows")'.dependencies]
 schannel = "0.1.16"
 
-[target.'cfg(not(any(target_os = "windows", target_os = "macos", target_os = "ios")))'.dependencies]
+[target.'cfg(feature = "mbedtls")'.dependencies]
```

Comment:
You can't cfg on features in the dependency section.

If this is designed for SGX, should it just be filtered to that target_os?